src/ui/scss/component/_media.scss

```diff
@@ -260,11 +260,16 @@
 
 .media__info-text {
   font-size: 1.15rem;
+  word-break: break-word;
 
   &:not(:last-of-type) {
     margin-bottom: var(--spacing-vertical-large);
   }
 
+  &.media__info-text--small {
+    max-width: 70%;
```

Comment:
@seanyesmunt let me know if you have a specific value in mind :v:

Reply:
50rem seems to work pretty well.

Reply:
Ok done!